scripts/set_version.py

```diff
@@ -3,9 +3,27 @@
 """
 
 from argparse import ArgumentParser, RawDescriptionHelpFormatter
+import yaml
+
 
 QUARTO_TEMPLATE = '_quarto.yml.template'
-VARS_TEMPLATE = '_variables.yml.template'
+
+LANG_VARS_SOURCE = 'variables.yml'
```

Comment:
Confusing name?  Maybe `input_variables.yml`?

Reply:
Renamed it to `text_variables.yml`.